packages/server/src/payloads.ts

```diff
@@ -125,7 +126,13 @@ const getArticleEpicTests = async (
         const hardcodedTests = enableHardcodedEpicTests ? hardcodedEpicTests : [];
 
         if (isForcingTest) {
-            return [...hardcodedTests, ...regular, ...holdback, fallbackEpicTest];
+            return [
+                inEpicPaymentTestDraft,
```

Comment:
By putting it here, it's only possible to view the draft test by providing the `force-epic` query param